python/psqlpy/_internal/__init__.pyi

```diff
@@ -321,6 +321,7 @@ class Transaction:
         self: Self,
         querystring: str,
         parameters: list[Any] | None = None,
+        prepared: bool = True,
```

Comment:
Also, i would move this setting onto the pool definition
I think, you either want to use prep stmts everywhere, or nowhere
This is discussable

Reply:
I don't think so, the prepared statement can be used to prepare `SELECT, INSERT, UPDATE, DELETE, MERGE, or VALUES` statements, there are a lot of other stmts, like CREATE or DROP which cannot be executed.

So the user would set global `prepared = False` only for one CREATE or DROP statement for the whole service.